framework/OpenMod.Core/Helpers/ArgumentsParser.cs

```diff
@@ -42,11 +42,21 @@ public static string[] ParseArguments(string line)
                     case '"':
                         inQuote = !inQuote;
                         break;
-                    case ' ' when !inQuote:
+
+                    case ' ':
+                        if (inQuote)
+                        {
+                            currentArg.Append(argsBuilder[i]);
+                            continue;
+                        }
                         args.Add(currentArg.ToString());
                         currentArg.Clear();
                         break;
+
                     default:
+                        if (char.IsWhiteSpace(argsBuilder[i])) //Space was checked before
+                            continue;
+
```

Comment:
What does get fixed here? Why was argument parsing changed? Did you test if it correctly parses arguments with quotes and spaces?

Reply:
Yes, I tested it.
Argument parsing receive the 'full' command https://github.com/openmod/openmod/blob/277ba9f366ec6630753cef0af2cb40e279c41fb0/unturned/OpenMod.Unturned/OpenModUnturnedHost.cs#L186 and if i do `text = text.Where(c => !char.IsWhiteSpace(c)))` it will 'delete' all the spaces: `openmod plugins` -> `openmodplugins`

Reply:
I still dont get whzz this has to be done in ArgumentParser

Reply:
Reverted